diff -urN pangox-compat-0.0.2-orig/configure pangox-compat-0.0.2/configure
--- pangox-compat-0.0.2-orig/configure	2012-10-16 19:46:59.000000000 -0600
+++ pangox-compat-0.0.2/configure	2018-01-05 01:36:36.082091302 -0700
@@ -2978,7 +2978,7 @@
 $as_echo "$as_me:${as_lineno-$LINENO}: checking for C compiler version" >&5
 set X $ac_compile
 ac_compiler=$2
-for ac_option in --version -v -V -qversion; do
+for ac_option in --version -v -V; do
   { { ac_try="$ac_compiler $ac_option >&5"
 case "(($ac_try" in
   *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
